src/der.rs

```diff
@@ -62,9 +62,9 @@ pub(crate) enum Tag {
     OctetString = 0x04,
     OID = 0x06,
     Enum = 0x0A,
-    UTF8String = 0x0C,
+    // UTF8String = 0x0C,
```

Comment:
I think since this enum already has gaps in the the tag value space it's OK to delete the unused entries rather than leave them commented out. WDYT?

Reply:
Cool, I wasn't sure what would be preferred. I can remove the unused tag variants.